compiler/damlc/tests/daml-test-files/Newtype.daml

```diff
@@ -1,6 +1,7 @@
 -- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
+-- @INFO Eta reduce
```

Comment:
I’m not really a fan of the `eta reduce` hint. I think it is more confusing than helpful. Can we disable it by default? cc @hurryabit

Reply:
Done.

Reply:
I agree that's not a particularly helpful rule.